### PR TITLE
Add new goal template

### LINKED ISSUE #16: Create template for goals list page
- [x] Welcome message.
- [x] List title / question.
- [x] Add button.
- [x] List of goals.
- [x] Style the goals in the list  

Designing page for user story #3.

### PR DESCRIPTION
**Pull request by shiryz**
Adds the basic elements in the pages and styling.
Fixes #16.

### CODE REVIEW

**Comment by bradreeder** [comment]
@shiryz @emilyb7 Remember to add references to the issue your commit resolves in the commit message. see https://github.com/CYPIAPT-LNDSE/sdm-app/pull/12#issuecomment-276417635

If you do this for every commit it means all your issues will contain complete histories of the work that has been done on them. It makes it much simpler to track progress. :)

File: public/styles/_add-goal.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+
+.back-button {
+  margin-left: 5px;
+  flex-direction: row;
+  // height: 28px;
+  background-color: Transparent;
+  border: none;
+  text-shadow: 0 2px 1px #888;
```
Comment (des-des) [inline public/styles/_add-goal.scss]: I like to keep all my colours in named vars!

File: public/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
+.page, .add-goal-page {
```
Comment (des-des) [inline public/styles/style.css]: See: https://github.com/CYPIAPT-LNDSE/goals-app/pull/26/files/b5a467f2f1760e4c955742e58e23d2f06beac7f2#diff-8184fba094481277f55710a8ec54d415

image

**Comment by shiryz** [comment]
@des-des , are you happy to merge now?

**Comment by des-des** [comment]
@shiryz also assign me if you want me to merge!

**Comment by des-des** [comment]
@shiryz also see brads message ^^

image

**Pull request merged**